sample-hip/specs/hip-facade.yaml

```diff
@@ -25,15 +25,15 @@ paths:
         required: true
       responses:
         '200':
-          description: Successfully added patients
+          description: OK
           content:
             application/json:
               schema:
-                type: string
+                $ref: '#/components/schemas/FacadeResponse'
```

Comment:
I see `FacadeResponse` being reused for all APIs, can we have a `GenericResponse` to respond `acknowledgement` or `errors` and for cases like fetching `RequestStatuse` we can extend it and add the relevant fields.

Reply:
made changes, thanks